### ISSUE TITLE
KinematicBody2D spanning 3 collision blocks causes Y jitter

### ISSUE THREAD

**Issue by chucklepie** [godotengine/godot#40498]
**Godot version:**
3.2.2 stable (on linux, gles3)

**Issue description:**
I have a tilemap and a kinematicbody2d. The code, as below, simply applies a downward y velocity and calls move and slide. When walking y velocity is 0. However, when the kinematic body spans 3 collision tiles then every other frame changes y velocity toggles between 0 and 5.166667 continuously. As soon as the kinematic body only spans 2 tiles y becomes 0.

Image below shows the body spanning 3 tiles and as stated y jitters in this position.

![image](https://user-images.githubusercontent.com/12863685/87861868-6489d300-c942-11ea-84ff-5a9c193dfae9.png)

The code is as follows, inside _physics_process:

```
func move_player(delta):
	velocity.y+=Globals.GRAVITY*delta
	velocity.x=lerp(velocity.x,MAX_SPEED*direction,_get_h_weight())
	velocity=move_and_slide(velocity,Globals.UP)

```

I did notice a similar fault raised, but this is for a 3D object
https://github.com/godotengine/godot/issues/35780

**Comment by Calinou** [comment]
@chucklepie Please upload a [minimal reproduction project](https://github.com/godotengine/godot/blob/master/CONTRIBUTING.md#provide-a-simple-example-project) to make this easier to troubleshoot.

**Comment by pouleyKetchoupp** [comment]
Fixed in #42574, closing.
